apps/web/src/App/Pages/Auth/SignUp/index.tsx

```diff
@@ -1,5 +1,141 @@
-import { FC } from 'react'
+import { FC, SyntheticEvent, useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import {
+  Container,
+  Card,
+  CardHeader,
+  Heading,
+  CardBody,
+  Stack,
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  FormErrorMessage,
+} from '@chakra-ui/react'
+import { Link, useNavigate } from 'react-router-dom'
+import { SmallAddIcon } from '@chakra-ui/icons'
+
+import { useLocalization } from '../../../../hooks/useLocalization.ts'
```

Comment:
Поправь импорты: должно быть без .ts (как в остальных местах)
```suggestion
import { useLocalization } from '../../../../hooks/useLocalization'
```